cli/tools/test_runner.rs

```diff
@@ -89,6 +89,14 @@ pub fn render_test_file(
     options
   ));
 
+  test_file.push_str(
+    r"
+if ('close' in globalThis) {
+  globalThis.close();
+}
+    ",
+  );
```

Comment:
I don't think this is correct; tests are not run in worker context

Reply:
Wasn't sure if unit tests invoked the manifest or not (seemingly not).